Compiler throws warning on compilation

warns that Bonsai.Design cannot be found:
```
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1820,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "Bonsai.Design". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.
```

--- a/Bonsai.Miniscope/Miniscope.csproj
+++ b/Bonsai.Miniscope/Miniscope.csproj
@@ -34,9 +34,6 @@
       <HintPath>..\packages\Bonsai.Core.2.3.0\lib\net45\Bonsai.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Bonsai.Design">
-      <HintPath>..\..\..\bonsai\Bonsai.Design\obj\Debug\Bonsai.Design.dll</HintPath>
-    </Reference>
     <Reference Include="OpenCV.Net">
       <HintPath>..\packages\OpenCV.Net.3.3.1\lib\net40\OpenCV.Net.dll</HintPath>
     </Reference>

